python_ta/reporters/templates/stylesheet.css

```diff
@@ -1,19 +1,15 @@
-body {
```

Comment:
Please leave the body. You can add an additional class if needed.

Regarding the outdated, my mistake, it looks like it has not been merged, https://github.com/pyta-uoft/pyta/pull/304